modules/running-insights-operator-gather-cli.adoc

```diff
@@ -13,37 +13,50 @@ You can run an Insights Operator gather operation by using the {product-title} c
 * You are logged in to {product-title} as a user with the `cluster-admin` role.
 
 .Procedure
-* Enter the following command to run the gather operation:
-+
-[source,terminal]
-----
-$ oc apply -f <your_datagather_definition>.yaml
-----
-+
-Replace `<your_datagather_definition>.yaml` with a configuration file that contains the following parameters:
+
+. Create a YAML file with the following `DataGather` specification:
 +
 [source,yaml]
 ----
-apiVersion: insights.openshift.io/v1alpha1
+apiVersion: insights.openshift.io/v1alpha2
 kind: DataGather
 metadata:
   name: <your_data_gather> <1>
 spec:
  gatherers: <2>
```

Comment:
This is not a correct structure, it should look like this:

```bash
  gatherers:
    mode: Custom
    custom:
      configs:
        - name: workloads
          state: Disabled
````
Another mode option is `All` that will basically run all the gatherers without any limitation

Reply:
Fixed. Rewrote the entire procedure accordingly.

FYI - Note: We are moving our content source from AsciiDoc to DITA, sometime in the final quarter (date TBC). In prep for that change, I removed the annotation numbering in the code snippets for all procedures "touched" in this PR. This is because this is no longer supported when we move. The notes and restructuring should hopefully enhance too. Thanks :-)